rbd.c

```diff
@@ -789,6 +895,41 @@ static int tcmu_rbd_lock(struct tcmu_device *dev, uint16_t tag)
 	if (ret)
 		goto done;
 
+#if !defined LIBRADOS_SUPPORTS_GETADDRS && defined RBD_LOCK_ACQUIRE_SUPPORT
```

Comment:
Nit: I am still voting for this code to be removed. The `LIBRADOS_SUPPORTS_GETADDRS` support will be backported through nautilus so for the sake of code-clarity, we shouldn't have to worry about alternative approached.